jwst/pipeline/calwebb_spec3.cfg

```diff
@@ -0,0 +1,18 @@
+name = "Spec3Pipeline"
+class = "jwst.pipeline.Spec3Pipeline"
+suffix = 'x1d'
+save_results = True
+
+    [steps]
+      [[skymatch]]
+        suffix = 'skymatch'
+      [[outlier_detection]]
+        suffix = 'crj'
```

Comment:
Looks like the AA is recommending the suffix "cr**f**" (cosmic ray **flagged**) for this type of product.